index.js

```diff
@@ -1,6 +1,6 @@
 module.exports = [
-  "Chrome >= 75",
-  "Edge >= 17",
-  "Firefox >= 70",
-  "Safari >= 12",
+  "Chrome >= 80",
+  "Edge >= 80",
+  "Firefox >= 75",
+  "Safari >= 13.1",
```

Comment:
Safari 13.1 is less than 18 months old and Safari doesn't auto-update 🙁 Is there a reason to drop support for 12?

Reply:
Updated to Safari 13